index.js

```diff
@@ -1,8 +1,26 @@
 'use strict';
 
 module.exports = {
-	extends: 'stylelint-config-suitcss',
+	extends: ['stylelint-config-standard-scss', 'stylelint-config-suitcss'],
 	rules: {
+		// redeclared from standard-scss because suitcss sets to `always`
+		'block-closing-brace-newline-after': [
+			'always',
+			{
+				ignoreAtRules: ['if', 'else'],
+			},
+		],
+		// this is being set in stylelint-standard, but we don't want it
+		'declaration-empty-line-before': null,
+		// our rules from here on
+		'alpha-value-notation': null, // not ready for this syntax yet
+		'at-rule-disallowed-list': [
+			['extend', 'import'],
+			{
+				severity: 'error',
+				message: 'Prefer @use and @forward rather than @import.',
+			},
+		],
```

Comment:
Should this be noted in the README, changelog and/or tests?

Reply:
Updated!